data_diff/diff_tables.py

```diff
@@ -73,11 +76,23 @@ def __post_init__(self):
 
     @property
     def _key_column(self):
-        return ColumnName(self.key_column)
+        return self._quote_column(self.key_column)
 
     @property
     def _update_column(self):
-        return ColumnName(self.update_column)
+        return self._quote_column(self.update_column)
+
+    def _quote_column(self, c):
+        if self.quote_columns:
+            return self.database.quote(c)
+        return c
```

Comment:
<img width="1322" alt="CleanShot 2022-06-14 at 16 38 49@2x" src="https://user-images.githubusercontent.com/97400/173684315-c79d619e-ca0d-4951-87c8-0e32c9c5e5b7.png">
<img width="406" alt="CleanShot 2022-06-14 at 16 39 05@2x" src="https://user-images.githubusercontent.com/97400/173684356-889d3e2e-df1f-4104-8a62-c096c19ec2a1.png">

Something's not right. I get an error pretty much no matter what I do here.

Reply:
These IDs are actual strings.. We don't support that yet.